integration/testutil.go

```diff
@@ -521,7 +531,7 @@ func newSnapshotterBaseShell(t *testing.T) (*shell.Shell, func() error) {
 	}
 	sh := shell.New(de, testutil.NewTestingReporter(t))
 	if !isTestingBuiltinSnapshotter() {
-		if err := testutil.WriteFileContents(sh, defaultContainerdConfigPath, []byte(proxySnapshotterConfig), 0600); err != nil {
```

Comment:
Is the containerd `proxy_plugins.proxy_plugins.soci` configuration no longer necessary?